docs/vaahstore/index.md

```diff
@@ -0,0 +1,63 @@
+# VaahStore
+
+[[toc]]
+
+## Introduction
+
+VaahStore is an E-commerce module for `VaahCMS` build on top of latest opensource technologies such as [laravel 10](https://laravel.com/docs/10.x) and [Vue 3](https://vuejs.org/guide/introduction.html).
+
+It is suitable for all small or large E-Commerce business demands using a simple set-up procedure. Built on top of `VaahCms` and equipped with an easy product information management. Some features are as follows:
+
+- Multi-vendor <sub>(able to add any vendor to individual store)</sub>
+- Multi-currency <sub>(support multiple currency's)</sub>
+- Multi-lingual <sub>(support multiple language's)</sub>
+- Multi Country Taxes <sub>(can handle multiple taxes)</sub>
+- Wishlists
+- Headless Product Content <sub>(data by API's)</sub>
+- Waitlist
+- Brand Store <sub>(create own brand)</sub>
+- Abandoned Cart <sub>(forbidden cart)</sub>
+- Coupon
+- AR Products Assets <sub>(view and add 3D products)</sub>
+- Subscriptions
+- Dropshipping
+- Affiliates
+
+### Why VaahStore ?
+
+- It is completely opensource.
+- It is based on latest and stable versions.
+- It is suitable for small to large individual or companies.
+- It can be customizable.
+
+## Requirements
+
+### Minimum System Requirements
+
+::: tip requirements
+- [npm](https://docs.npmjs.com/cli)
+- [VaahCMS](https://www.npmjs.com/package/vaah)
+- php >= 7.4
```

Comment:
@bharatkumar-j001 this is also incorrect if laravel 10 has a minimum requirement of PHP 8.1 then and if this module is built on laravel 10, then how can it run on PHP 7.4??